src/models/client.ts

```diff
@@ -205,32 +231,60 @@ export class ClientEnvironment {
     */
   }
 
-  public async clearBrowserCache() {
-    await this.network.clearBrowserCache();
+  private async buildCapturedResponse(response: Network.ResponseReceivedParameters): Promise<CapturedResponse> {
+    return {
+      responseMeta: response.response,
+      body: (await this.network.getResponseBody({requestId: response.requestId})).body
+    };
+  }
+
+  /**
+   * Load page in active tab. Resolves on HTML load. To wait for a full page load, and
+   * also recieve all response data, use load() method instead.
+   * @return HTML response and page frame data
+   */
+  public async navigate(arg?: string | NavigateOptions): Promise<PageNavigateResult> {
+    const {page, frame} = await this.privateNavigate(false, arg);
+    return {page, frame};
+  }
+
+  /**
+   * Load page in active tab. Resolves on page load complete, i.e. Max(onLoad, all responses recieved)
```

Comment:
I think we should specifically callout that "page load complete" means waiting for the `load` event.

Reply:
I'll expand out the existing `i.e. Max(onLoad, all responses received` into a full sentence. Also you are correct that it is `load` event, not the `onLoad` event. `onLoad` is technically a method which is overridable if you want to execute code on the `load` event. It's so common to swap the two that I had never considered the difference before.